README.md

```diff
@@ -8,31 +8,42 @@ It logs the information of the incoming requests, validates their payload, and s
 3. build the app:<br/>
- `yarn build`
+   `yarn build`
 4. set the required environment variables described in the related section
 5. start the server:<br/>
-  `yarn start`
+   `yarn start`
+
+### Environment variables
 
-###Environment variables
 The table lists the environment variables needed by the application, they must be customized as needed.
 For your convenience a `env.example` is provided: you can copy it to an `.env` file and then change the variables with your values,
 the app will load them on start.
 
-| Variable name                          | Description                                                                       | type    |
-| -------------------------------------- | --------------------------------------------------------------------------------- | ------- |
-| BACKEND_PORT                           | Port of Express server                                                            | number  |
-| DB_SOURCE                              | The path of the JSON file used to store the data                                  | number  |
-| EMAIL_PASSWORD                         | The password used to connect to the email server                                  | string  |
-| EMAIL_USER                             | The user to connect to the email server                                           | string  |
-| EMAIL_SMTP_HOST                        | The hostname or IP address of the SMTP server                                     | string  |
-| EMAIL_SMTP_PORT                        | The port of the SMTP server                                                       | int     |
-| EMAIL_SMTP_SECURE                      | If set to true, uses TLS in SMTP connection                                       | boolean |
-| EMAIL_SENDER                           | The email address of the sender                                                   | string  |
-| EMAIL_RECIPIENT                        | The email address of the recipient                                                | string  |
+| Variable name     | Description                                      | type    |
+| ----------------- | ------------------------------------------------ | ------- |
+| BACKEND_PORT      | Port of Express server                           | number  |
+| DB_SOURCE         | The path of the JSON file used to store the data | number  |
+| EMAIL_PASSWORD    | The password used to connect to the email server | string  |
+| EMAIL_USER        | The user to connect to the email server          | string  |
+| EMAIL_SMTP_HOST   | The hostname or IP address of the SMTP server    | string  |
+| EMAIL_SMTP_PORT   | The port of the SMTP server                      | int     |
+| EMAIL_SMTP_SECURE | If set to true, uses TLS in SMTP connection      | boolean |
+| EMAIL_SENDER      | The email address of the sender                  | string  |
+| EMAIL_RECIPIENT   | The email address of the recipient               | string  |
+
+## Postman Collection Tests
```

Comment:
Please add the information that it is required that the app must be running before the execution of this test suite.

Reply:
thanks ;)